src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -27,6 +27,11 @@
         <target state="translated">Akce nemůžou zůstat prázdné.</target>
         <note />
       </trans-unit>
+      <trans-unit id="Add_to">
+        <source>Add to</source>
+        <target state="new">Add to</target>
```

Comment:
not legal.  this should be `Add to '{0}'`

Reply:
Note "add to" sounds wrong as well.  I don't really have an idea waht it means (for example: will it leave the member alone where i am?  or does it pull it up?).  The feature is called "Pull members up" this text should be `Pull '{0}' up to '{1}'`